### PR TITLE
XUnit XML File Reporter

### PR DESCRIPTION
**Pull request by sebastianneubert**

### CODE REVIEW

**Comment by digitalkaoz** [comment]
can you squash the commits?

File: src/Reporter/XUnitReporter.php [hunk 1]
```diff
@@ -26,6 +26,64 @@ public function process($result)
...
+        $testSuite->setAttribute('name', reset($this->results)['parent']);
+        $testSuite->setAttribute('tests', count($this->results));
+        $testSuite->setAttribute('assertions', count($this->results));
+        $testSuite->setAttribute('failures', $failures);
+        $testSuite->setAttribute('errors', '0');
+        $testSuite->setAttribute('time', '0.5');
+
+        $xml->save($this->filename);
+
+        echo 'Writing XUnit Output to file: ' . $this->filename;
```
Comment (digitalkaoz) [inline src/Reporter/XUnitReporter.php]: dont echo, inject the command output instance

**Comment by phmLabs** [comment]
As I need the sources I will merge the request without squashing. Next time it will be done ;)

**Comment by digitalkaoz** [comment]
omg, try to find bugs with git bisect when you have such a crippled commit history ;)

**Pull request merged**